src/components/CoinList.tsx

```diff
@@ -95,17 +86,29 @@ export default function CoinList(props: CoinListProps) {
     {
       accessorKey: 'coin_id',
       size: 100,
-      header: ({ column }) => (
+      header: () => (
         <Button
           className='px-0'
           variant='link'
-          onClick={() => column.toggleSorting(column.getIsSorted() === 'asc')}
+          onClick={() => {
+            if (props.sortMode === 'parent_coin_id') {
```

Comment:
I would expect this to sort by the current coin id, not the parent coin id. But still, I think it'd be fine if we just make this column non-sortable personally.

Reply:
yeah - sorting on coin id is random. switched it to keep current functionality. lemme know if you want to remove sorting on this column completely. im ambivalent :)